Test04.pv

```diff
@@ -3,7 +3,7 @@ background(1)
 
 import json
 
-with open('crappydesign_scraped.json') as infile:
+with open('crappydesign_scraped02.json') as infile:
```

Comment:
I didn't have the file `crappydesign_scraped.json` on my machine, so I took a different one